doc/topics/cloud/aws.rst

```diff
@@ -721,6 +721,24 @@ them have never been used, much less tested, by the Salt Stack team.
 .. __: https://aws.amazon.com/marketplace
 
 
+NOTE: If ``image`` of a profile does not start with ``ami-``, latest
+image with that name will be used.
```

Comment:
Could we add an example of this in the documentation just so it's extremely clear?

Reply:
If we can get these requested docs added, then I think we can get this merged.